aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,301 @@
 
-.. _issue: https://github.com/aspnet/Docs/issues/101
+This article explains and shows how to use the bundling and minification features included with the ASP.NET 5 Web Application template in `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=517106>`_. 
+
+Overview
+--------
+Because bundling combines multiple files into a single file, it reduces the number of requests to the server that is required to retrieve and display a web asset, such as a web page. You can create CSS, JavaScript and other bundles. Fewer files, means fewer HTTP requests from your browser to the server or from the service providing your application. This results in improved first page load performance.
+
+Minification performs a variety of different code optimizations to reduce the size of requested assets (such as CSS, image, JavaScript files). Common results of minification include removing unnecessary white space and comments, and shortening variable names to one character. 
+
+Consider the following JavaScript function:
+
+.. code-block:: javascript
+
+	AddAltToImg = function (imageTagAndImageID, imageContext) {
+		///<signature>
+		///<summary> Adds an alt tab to the image
+		// </summary>
+		//<param name="imgElement" type="String">The image selector.</param>
+		//<param name="ContextForImage" type="String">The image context.</param>
+		///</signature>
+		var imageElement = $(imageTagAndImageID, imageContext);
+		imageElement.attr('alt', imageElement.attr('id').replace(/ID/, ''));
+	}
+
+After minification, the function is reduced to the following:
+
+.. code-block:: javascript
+
+	AddAltToImg=function(t,a){var r=$(t,a);r.attr("alt",r.attr("id").replace(/ID/,""))};
+	
+In addition to removing the comments and unnecessary whitespace, the following parameters and variable names were renamed (shortened) as follows:
+
+==================  =======  
+Original            Renamed      
+==================  =======  
+imageTagAndImageID  t  
+imageContext        a  
+imageElement        r   
+==================  =======  
+
+Bundling and Minification in Visual Studio using Gulp
+-----------------------------------------------------
+You may be familiar with *runtime* bundling and minification using ASP.NET Web Optimization. In ASP.NET 5, you can bundle and minify your client-side resources during *design-time*. Using design-time bundling and minification, you build your minified files up front and then deploy them with your app as static files. By bundling and minifying up front, you have the advantage of fewer moving parts and reduced server load. However, it's important to recognize that design-time bundling and minification increases build complexity and only work with static files.
```

Comment:
Do you mean server side B&M from the previous version? clarify that and maybe link to my article. Any client side you're not doing B&m at design time, your doing it as a part of the build process with build events. - Avoid the cliche *moving parts* - actually, there are more moving parts with client side. Furthermore, the server side B&M is a one time cost. Does old school B&M work with dynamic files?